lib/Runtime/ByteCode/OpCodeUtil.cpp

```diff
@@ -161,6 +162,7 @@ namespace Js
         {
             Assert(false);
             __assume(false);
```

Comment:
Unreachable code warnings are probably because this line makes it so that control flow can't reach the following line.